CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Change log
 
-All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
+All notable changes to this project will be documented in this file.
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) 
+and this project adheres to [Semantic Versioning](http://semver.org).
+
+## 2020-3-20 - Release - 2.2.0
```

Comment:
I think the changes under 2.1.0 and 2.0.2 are unreleased, so we should include them all under the 2.2.0 notes.

I've been trying to add to changelog with each PR we accept, but I have some leftover that need to be add ed from an unfinished release prep branch here:

https://github.com/puppetlabs/puppet_metrics_dashboard/compare/master...suckatrash:release_prep

Reply:
I've moved the above, and added the leftovers from your branch!